client/js/components/heading.js

```diff
@@ -2,17 +2,27 @@ import React from 'react'
 import injectSheet from 'react-jss'
 import classNames from 'classnames'
 
-const Heading = ({ classes, text }) => {
-	return <div className={classes.headingText}>{text}</div>
+const Heading = ({ classes, text, size }) => {
+	return <div className={classes[`${size}`]}>{text}</div>
 }
 
 const styles = {
-	headingText: {
+	medium: {
 		color: '#D8D8D8',
 		'font-size': '20px',
 		'font-weight': 'bold',
 		'letter-spacing': '3.73px',
 		'line-height': '24px',
+		'justify-self': 'start',
+	},
+	small: {
+		height: '17px',
+		width: '78px',
```

Comment:
We don't actually need the width here. It's listed in InVision, but it's not needed 😄